io/io/inc/ROOT/RIoUring.hxx

```diff
@@ -56,6 +61,91 @@ public:
       static const bool available = RIoUring::CheckIsAvailable();
       return available;
    }
+
+   /// Access the raw io_uring instance.
+   struct io_uring *GetRawRing() {
+      return &fRing;
+   }
+
+   /// Basic read event composed of RIOVec IO data and a target file descriptor.
+   struct RReadEvent {
+      RRawFile::RIOVec* fIoVec = nullptr;
```

Comment:
With this we have a circular dependency between `RRawFile` and `RIoUring`.  I think I'd prefer to `RReadEvent` to have it's own version of the offset, size, outsize, and buffer members.

Reply:
Thank you, done in 592ef14